src/ci/docker/run.sh

```diff
@@ -205,7 +205,7 @@ mkdir -p $objdir/tmp
 mkdir -p $objdir/cores
 mkdir -p /tmp/toolstate
 
-args=
+args=" --env RUST_BACKTRACE"
```

Comment:
It would be nicer to add it here: https://github.com/rust-lang/rust/blob/31e3f76b95d9b272e35b34a7f0e3f9426efc373f/src/ci/docker/run.sh#L313-L335

so it groups up with the other explicit variables.

Reply:
Done.